086-monitoring-of-custom-resources.md

```diff
@@ -0,0 +1,61 @@
+# Monitoring of custom resources
+
+In https://github.com/strimzi/strimzi-kafka-operator/issues/9802 there was a discussion about how to monitor the state of a custom resources (CR) like `KafkaTopic`, `KafkaUser`, ... .
+For a simple and Kubernetes native monitoring, [kube-state-metrics](https://github.com/kubernetes/kube-state-metrics) (KSM) can be used which utilizes e.g. the status object of a Kubernetes resource.
+
+## Current situation
+
+Strimzi provides metrics for several use cases but currently not for the state of `KafkaTopic` and `KafkaUser` CRs.
+TODO need input for the status of other CRs and their metrics, e.g. `Kafka`, `StrimziPodSet`, `KafkaRebalance`...
+
+## Motivation
+
+Strimzi already provides example configurations being compatible with the prometheus-operator via e.g. `PodMonitor` for scraping the metrics and e.g. `PrometheusRules` for alerting.
+This is implemented by metrics-exporter via JMX or kafka-exporter.
```

Comment:
Actually, as we are talking about operators here, they handle it on their own. So I would probably delete this sentence.